app/components/Account/AccountSelector.jsx

```diff
@@ -94,15 +105,189 @@ class AccountSelector extends React.Component {
         }
     }
 
+    _addToIndex(accountName, noDelay = false) {
```

Comment:
Please see my change above. Now test this and put `noDelay` to `true`. 

The AccountSelector breaks due to infinity loop, which should not happen. This indicates the rendering / lazy loading is not coupled properly. Please investigate

Reply:
Will look deeper into this, sorry for the miss.